orchestrator/lib/coolfhir/util.go

```diff
@@ -399,3 +401,29 @@ func ContainsCoding(code fhir.Coding, concepts ...fhir.CodeableConcept) bool {
 	}
 	return false
 }
+
+func GetTaskByIdentifier(ctx context.Context, fhirClient fhirclient.Client, identifier fhir.Identifier) (*fhir.Task, error) {
```

Comment:
would be good to have a unit test. You can use `StubFHIRClient` for easy testing.

Reply:
Yeah I agree, tried it before, but ran into a circular dependency. It is currently indirectly tested twice in the demo and zpf service. I'll try and fix it later (after I solved the Practitioner logic from ZPF - higher prio issue).